File: src/components/DoubleString.java
package components;

public class DoubleString {
	public String writer;
	public String shower;
	
	public String getWriter() {
		return writer;
	}
	
	public void setWriter(String writer) {
		this.writer = writer;
	}
	
	public String getShower() {
		return shower;
	}
	
	public void setShower(String shower) {
		this.shower = shower;
	}
	
	public DoubleString(String writer, String shower) {
		setWriter(writer);
		setShower(shower);
	}
	

}

File: Project/doublestring.cpp
#include "doublestring.h"

DoubleString::DoubleString(QString writer, QString shower)
{
    this->shower = shower;
    this->writer = writer;
}

DoubleString::DoubleString()
{
    this->shower = "";
    this->writer = "";
}

QString DoubleString::getShower() const
{
    return shower;
}

void DoubleString::setShower(const QString &value)
{
    shower = value;
}

QString DoubleString::getWriter() const
{
    return writer;
}

void DoubleString::setWriter(const QString &value)
{
    writer = value;
}
